type-tests/preview/@ember/owner-tests.ts

```diff
@@ -35,19 +41,19 @@ aFactory.create({
   hasProps: false,
 });
 
-// These should be rejected by way of EPC
-// @ts-expect-error
+// NOTE: it would be nice if these could be rejected by way of EPC, but alas: it
```

Comment:
“Excess Property Checking”: the feature where if you pass an object *literal* as an argument with extra properties than the parameter type specifies, TS will error since that is basically guaranteed to e incorrect.